### PR TITLE
feat(run): add requester id in list pipeline/model run response

### PR DESCRIPTION
**Pull request by joremysh**
Because

- runs pages need to display credit owner field

This commit

- expose requester id in response
- rename rpc name and add documentations (from previous TODOs in https://github.com/instill-ai/protobufs/pull/472)

### CODE REVIEW

File: model/model/v1alpha/model.proto [hunk 1]
```diff
@@ -1770,6 +1770,11 @@ message ModelRun {
     (google.api.field_behavior) = OUTPUT_ONLY,
     (google.api.field_behavior) = OPTIONAL
   ];
+  // Requester ID. If current viewing requester does not have enough permission, it will return null.
```
[the remark this answers is not in the document]
Comment (joremysh) [reply model/model/v1alpha/model.proto:1773]: just checked again with the code and previous discussion. It seems there is no scenario that requester id will be hidden / anonymised while the runs are accessible by the user.

The only case I can think of now is: if some org are deleted, CheckNamespaceByUIDAdmin cannot return the requester id.

File: model/model/v1alpha/model_public_service.proto [hunk 2]
```diff
@@ -737,7 +737,7 @@ service ModelPublicService {
   //
   // Returns a paginated list of runs for 1 or more models. This is mainly used by credit dashboard.
   // The requester can view all the runs that consumed their credits across different models.
```
[the remark this answers is not in the document]
Comment (joremysh) [reply model/model/v1alpha/model_public_service.proto:739]: updated

File: model/model/v1alpha/model.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (jvallesm) [reply model/model/v1alpha/model.proto:1773]: > The only case I can think of now is: if some org are deleted, CheckNamespaceByUIDAdmin cannot return the requester id.

In this case, even if we soft-delete the user / org and we can access the requester ID, we shouldn't show it, as the ID could have been taken by another namespace. Also, for data protection reasons we might want to show `(deleted user)` even if the requester ID hasn't been taken.

In any case, the point here is that the documentation on the field doesn't reflect how the endpoint will behave (either the client can't see the run, or they will be able to see the requester ID).

File: model/model/v1alpha/model_public_service.proto (hunk 2 above)
[the remark this answers is not in the document]
Comment (jvallesm) [reply model/model/v1alpha/model_public_service.proto:739]: 🤔 is it?

File: vdp/pipeline/v1beta/pipeline.proto [hunk 3]
```diff
@@ -1880,8 +1880,8 @@ message ListPipelineRunsRequest {
   optional string order_by = 7 [(google.api.field_behavior) = OPTIONAL];
 }
 
-// ListPipelineRunsByCreditOwnerRequest is the request message for ListPipelineRunsByCreditOwner.
-message ListPipelineRunsByCreditOwnerRequest {
+// ListPipelineRunsByRequesterRequest is the request message for ListPipelineRunsByRequester.
+message ListPipelineRunsByRequesterRequest {
```
[the remark this answers is not in the document]
Comment (joremysh) [reply vdp/pipeline/v1beta/pipeline.proto:1884]: updated

[the remark this answers is not in the document]
Comment (jvallesm) [reply vdp/pipeline/v1beta/pipeline.proto:1884]: 👍 thanks

**Review by jvallesm — APPROVED** [review]

**Pull request merged**